site/cni-plugin.md

```diff
@@ -1,13 +1,14 @@
 ---
-title: Integrating Kubernetes via the CNI Plugin
+title: Weave CNI Plugin
```

Comment:
@abuehrle I thought we wanted all headings to be "actions". Also, I am concerned that w/o k8s/mesos being mentioned in the heading, folks really won't know know what this is about - 'CNI' is not a well-known term.

So I'd opt for 'Integrating Kubernetes and Mesos via the CNI Plugin', but that's rather long.